packages/monaco/src/browser/monaco-theme-service.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject, named } from "inversify";
+import { ContributionProvider } from "@theia/core";
+import { ThemeProvider } from "@theia/core/lib/common/theming-protocol";
+import { MonacoThemeProvider, MonacoTheme } from "../common/monaco-theme-protocol";
+import { Deferred } from "@theia/core/lib/common/promise-util";
+
+/**
+ * This class hooks onto the Theia Timport { error } from 'shelljs';
+heme service as a Theme provider, while it doesn't provide any theme.
```

Comment:
looks like something got deleted by accident

Reply:
Miss click with the middle mouse button under Ubuntu, it's driving me crazy.